README.md

```diff
@@ -38,6 +38,12 @@ Desde Visual Studio:
 4. Clic en la pestaña Examinar y busque `TransbankSDK`
 5. Clic en el paquete `TransbankSDK`, seleccione la versión que desea utilizar y finalmente selecciones instalar.
 
+Con .NET Core:
```

Comment:
This is called .Net CLI in NuGet

Reply:
So you suggest I phrase that as "Con .Net CLI:"?

Reply:
I think it sounds better, because .Net Core is more like the Framework, you can use the same command to install if you are using .Net Standard or .Net Framework also.

Reply:
Thanks, done! :)